apps/mnt/src/app/layout.tsx

```
@@ -11,18 +15,23 @@ export const metadata: Metadata = {
+
     return (
-        <html lang="en">
+        <html lang={locale ?? defaultLocale}>
             <body>
                 <Suspense>
-                    <RefineContext defaultMode={defaultMode}>
-                        <>{children}</>
-                    </RefineContext>
+                    <NextIntlClientProvider messages={messages} locale={locale}>
+                        <RefineContext defaultMode={defaultMode}>
+                            <>{children}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)